File: Home Work/9. singleNumber.cpp
#include<iostream>
#include<vector>
using namespace std;

int singleNumber(vector<int> &nums){
    int ans = 0;
    for(int i: nums){
        ans = ans ^ i;
    }
    return ans;
}

int main(){
    vector<int> nums = {4, 2, 1, 2, 1};
    cout << "Single Number is : " << singleNumber(nums) << endl;
    return 0;
}

File: Home Work/9. singleNumber.js
let myArr = [4, 2, 1, 2, 1];
let singNum = function singleNumber(nums){
    let ans = 0;
    for(let num of nums){
        ans = ans ^ num;
    }
    return ans;
}

console.log("Songle Number is : ", singNum(myArr));